cmd/ctr/commands/pprof/pprof.go

```diff
@@ -182,6 +182,7 @@ func httpGetRequest(client *http.Client, request string) (io.ReadCloser, error)
 	if err != nil {
 		return nil, err
 	}
+	defer resp.Body.Close()
```

Comment:
All the callers of this currently close the body themselves, as the body is returned from this function. Is this one needed?

Reply:
may it is no necessary to close body